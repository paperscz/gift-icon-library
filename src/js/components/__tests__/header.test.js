import React from 'react';
import { mount } from 'enzyme';

import Header from '../header';

describe('The header component', () => {
  let component;

  it('should render with no props', () => {
    component = mount(<Header />);
    expect(component).toMatchSnapshot();
  });

  it('should render with props', () => {
    component = mount(
      <Header
        name="Headline"
        subhead="Test Subhead"
      />
    );
    expect(component).toMatchSnapshot();
  });



});