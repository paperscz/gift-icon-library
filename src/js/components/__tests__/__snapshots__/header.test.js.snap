// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The header component should render with no props 1`] = `
<Header>
  <header
    className="header"
  >
    <div
      className="container"
    >
      <h1>
        <span />
         
      </h1>
    </div>
  </header>
</Header>
`;

exports[`The header component should render with props 1`] = `
<Header
  name="Headline"
  subhead="Test Subhead"
>
  <header
    className="header"
  >
    <div
      className="container"
    >
      <h1>
        <span>
          Headline
        </span>
         
        Test Subhead
      </h1>
    </div>
  </header>
</Header>
`;
